homeassistant/components/flipr/sensor.py

```diff
@@ -60,31 +61,16 @@ class FliprSensor(FliprEntity, SensorEntity):
     @property
     def name(self):
         """Return the name of the particular component."""
-        return f"Flipr {self.flipr_id} {SENSORS[self.info_type]['name']}"
+        return f"Flipr {self.flipr_id} {self.entity_description.name}"
 
     @property
     def native_value(self):
         """State of the sensor."""
-        state = self.coordinator.data[self.info_type]
+        state = self.coordinator.data[self.entity_description.key]
         if isinstance(state, datetime):
             return state.isoformat()
         return state
 
-    @property
-    def device_class(self):
-        """Return the device class."""
-        return SENSORS[self.info_type]["device_class"]
-
-    @property
-    def icon(self):
-        """Return the icon."""
-        return SENSORS[self.info_type]["icon"]
-
-    @property
-    def native_unit_of_measurement(self):
-        """Return unit of measurement."""
-        return SENSORS[self.info_type]["unit"]
-
     @property
     def device_state_attributes(self):
         """Return device attributes."""
```

Comment:
The `attribution` can probably be moved to `FilprEntity` as well.
```py
_attr_extra_state_attributes = {ATTR_ATTRIBUTION: ATTRIBUTION}
```

Reply:
OK moved it.